src/docs/layouts/shortcodes/content/setup.md

```diff
@@ -39,13 +41,22 @@ From `deploy/complete/helm-chart` directory:
     helm install mushop-utils setup \
       --namespace mushop-utilities
     ```
->  **OPTIONAL** The Jenkins automation server can be enabled by setting `jenkins.enabled` to `true` in `values.yaml` or by adding the command line flag `--set jenkins.enabled=true` in the `helm install` command above.
 
-  ```shell--helm3
-  helm install mushop-utils setup \
-    --namespace mushop-utilities \
-    --set jenkins.enabled=true
-  ```
+    > **OPTIONAL** If will be provisioning ATP, Stream and Object Storage with OCI Service Broker, you should enable the Service Catalog  `catalog.enabled` to `true` in `values.yaml` or by adding the command line flag `--set catalog.enabled=true` in the `helm install` command above.
```

Comment:
If you would like to use the OCI service broker to provision ATP, Stream and Object Storage, you should enable...

Reply:
😱